<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>595</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>595</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator by Fedor Sanin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>595</width>
     <height>720</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>595</width>
     <height>720</height>
    </size>
   </property>
   <widget class="QPushButton" name="pbt_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(81, 81, 81);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(81, 81, 81);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>480</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(81, 81, 81);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(81, 81, 81);</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(81, 81, 81);</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(81, 81, 81);</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(81, 81, 81);</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_8">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(81, 81, 81);</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_0">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>600</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(81, 81, 81);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(81, 81, 81);</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_pstvt_change">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>600</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(81, 81, 81);</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_dot">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>600</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(81, 81, 81);</string>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_equality">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>600</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(61, 61, 61);</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_minus">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(61, 61, 61);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_plus">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(61, 61, 61);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_mlt">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(61, 61, 61);</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_dvd">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(61, 61, 61);</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_c">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(61, 61, 61);</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_del">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(61, 61, 61);</string>
    </property>
    <property name="text">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_persent">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>150</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(61, 61, 61);</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lbl_number">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>0</y>
      <width>601</width>
      <height>121</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Microsoft YaHei Light&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
